src/UI/Article.jsx

```diff
@@ -75,7 +84,21 @@ function Article(props) {
         backgroundPosition: 'center',
       }}
     >
-      <div className="absolute bottom-0 left-0 right-0 h-1/2 bg-black/30 p-4 backdrop-blur-lg">
+      <a
+        href={getLink(props)}
+        target="_blank"
+        rel="noopener noreferrer"
+        className={
+          'absolute right-2 top-2 rounded-lg bg-white/[.50] object-right opacity-100'
```

Comment:
Just a few small changes here for more consistency:

To add: `p-1`

To update: `right-0 top-0 bg-black/60`

To remove: `rounded-lg`